includes/fraud-prevention/class-order-fraud-and-risk-meta-box.php

```diff
@@ -0,0 +1,129 @@
+				$icon_alt    = __( 'Green check mark', 'woocommerce-payments' );
+				$status      = __( 'No action taken', 'woocommerce-payments' );
+				$description = __( 'The payment for this order passed your risk filtering.', 'woocommerce-payments' );
+				echo '<p class="wcpay-fraud-risk-meta-allow"><img src="' . esc_url( $icon_url ) . '" alt="' . esc_html( $icon_alt ) . '"> ' . esc_html( $status ) . '</p><p>' . esc_html( $description ) . '</p>';
+				break;
+
+			case Fraud_Outcome_Status::REVIEW:
+				$icon_url        = plugins_url( 'assets/images/icons/shield-stroke-orange.svg', WCPAY_PLUGIN_FILE );
+				$icon_alt        = __( 'Orange shield outline', 'woocommerce-payments' );
+				$status          = __( 'Held for review', 'woocommerce-payments' );
+				$description     = __( 'The payment for this order was held for review by your risk filtering. You can review the details and determine whether to approve or block the payment.', 'woocommerce-payments' );
+				$callout         = __( 'Review payment', 'woocommerce-payments' );
+				$transaction_url = WC_Payments_Utils::compose_transaction_url( $intent_id, $charge_id );
+				echo '<p class="wcpay-fraud-risk-meta-review"><img src="' . esc_url( $icon_url ) . '" alt="' . esc_html( $icon_alt ) . '"> ' . esc_html( $status ) . '</p><p>' . esc_html( $description ) . '</p><a href="' . esc_url( $transaction_url ) . '" target="_blank" rel="noopener noreferrer">' . esc_html( $callout ) . '</a>';
+				break;
+
+			case Fraud_Outcome_Status::REVIEW_ALLOWED:
+				$icon_url        = plugins_url( 'assets/images/icons/check-green.svg', WCPAY_PLUGIN_FILE );
+				$icon_alt        = __( 'Green check mark', 'woocommerce-payments' );
+				$status          = __( 'Held for review', 'woocommerce-payments' );
+				$description     = __( 'Need text to display once allowed.', 'woocommerce-payments' );
+				$callout         = __( 'Review payment', 'woocommerce-payments' );
+				$transaction_url = WC_Payments_Utils::compose_transaction_url( $intent_id, $charge_id );
+				echo '<p class="wcpay-fraud-risk-meta-allow"><img src="' . esc_url( $icon_url ) . '" alt="' . esc_html( $icon_alt ) . '"> ' . esc_html( $status ) . '</p><p>' . esc_html( $description ) . '</p><a href="' . esc_url( $transaction_url ) . '" target="_blank" rel="noopener noreferrer">' . esc_html( $callout ) . '</a>';
+				break;
+
+			case Fraud_Outcome_Status::BLOCK:
+				// TODO: Are we going to cancel the orders? The note states we are, but if we do, we do not allow them to correct mistakes and could create a lot of additional orders.
+				$icon_url        = plugins_url( 'assets/images/icons/shield-stroke-red.svg', WCPAY_PLUGIN_FILE );
+				$icon_alt        = __( 'Red shield outline', 'woocommerce-payments' );
+				$status          = __( 'Blocked', 'woocommerce-payments' );
+				$description     = __( 'The payment for this order was blocked by your risk filtering, and the order has been cancelled.', 'woocommerce-payments' );
```

Comment:
Possible change to text:

> The payment for this order was blocked by your risk filtering. There is no pending authorization, and the order can be cancelled if you wish.

Reply:
I think we can remove the second sentence completely. No need to indicate, or suggest anything to the merchant. What do you think?